js2-mode.el

```diff
@@ -226,6 +226,13 @@ variable with predicate PRED."
   (make-variable-buffer-local name)
   (put name 'safe-local-variable pred))
 
+(defcustom js2-ignored-warnings nil
+  "A list of warning message types that will not be reported.
+
+Possible values are the keys of `js2-message-table'."
+  :group 'js2-mode
+  :type 'list)
```

Comment:
One way to review this kind of suggestion is to make the change and see if the `customize-variable` UI looks no worse (or probably better) than before.

Anyway, it must be fine.